[BUG] GitHub Actions / tox tests are failing

**Describe the bug**  
The GitHub actions resp. the `tox` test are failing already quite early and not related to formatting issues. 
The error message looks like this: 
```
  _ ERROR collecting .tox/py38-nocov-linux/lib/python3.8/site-packages/egon/data/airflow/dags/pipeline.py _
  .tox/py38-nocov-linux/lib/python3.8/site-packages/egon/data/airflow/dags/pipeline.py:381: in <module>
      foreign_lines = ElectricalNeighbours(
  .tox/py38-nocov-linux/lib/python3.8/site-packages/egon/data/datasets/electrical_neighbours.py:2217: in __init__
      super().__init__(
  <string>:7: in __init__
      ???
  .tox/py38-nocov-linux/lib/python3.8/site-packages/egon/data/datasets/__init__.py:277: in __post_init__
      last = list(self.tasks.last)[0]
  E   IndexError: list index out of range
```
On the powerd-data dev, the tests were running successfully. So it might be that the error is somehow related to [this commit](https://github.com/openego/eGon-data/commit/c9f72023671f919dda3cc8122cd0681ab44452fd) or a previous one that switches to the `wrapped_partial` way to create tasks in the `ElectricalNeighbours` dataset. 


---

--- a/src/egon/data/config.py
+++ b/src/egon/data/config.py
@@ -73,7 +73,8 @@ def settings() -> dict[str, dict[str, str]]:
                 "--jobs": 1,
                 "--random-seed": 42,
                 "--processes-per-task": 1,
-                "--scenarios": "status2023",
+                "--scenarios": ["status2019", "eGon2035"],
+                "--run-pypsa-eur": False,
             }
         }
     with open(files[0]) as f:

